### ISSUE TITLE
Many of our ESLint rules are not firing

### ISSUE THREAD

**Issue by bryanpizzillo** [NCIOCPL/clinical-trials-listing-app#33]
### Issue description
TL; DR; Most of our lint rules are not running on `npm lint` or on a compile.

See issue the issue details in https://github.com/NCIOCPL/cgov-react-app-playground/issues/25 for specifics.

> **ESTIMATE** TBD

**Comment by bryanpizzillo** [comment]
# Update Log
```
npm install -D --json @typescript-eslint/eslint-plugin@4.15.2 @typescript-eslint/parser@4.15.2 babel-eslint@10.1.0 eslint@7.20.0 eslint-config-airbnb@18.2.1 eslint-config-prettier@8.1.0 eslint-config-react-app@6.0.0 eslint-loader@4.0.2 eslint-plugin-flowtype@5.3.1 eslint-plugin-import@2.22.1 eslint-plugin-jsx-a11y@6.4.1 eslint-plugin-prettier@3.3.1 eslint-plugin-react@7.22.0 eslint-plugin-react-hooks@4.2.0 prettier@2.2.1
npm WARN deprecated eslint-loader@4.0.2: This loader has been deprecated. Please use eslint-webpack-plugin
npm WARN deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
npm WARN tsutils@3.20.0 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself.

{
  "added": [
    {
      "action": "add",
      "name": "ignore",
      "version": "4.0.6",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@eslint/eslintrc/node_modules/ignore"
    },
    {
      "action": "add",
      "name": "ms",
      "version": "2.1.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@eslint/eslintrc/node_modules/ms"
    },
    {
      "action": "add",
      "name": "debug",
      "version": "4.3.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@eslint/eslintrc/node_modules/debug"
    },
    {
      "action": "add",
      "name": "resolve-from",
      "version": "4.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@eslint/eslintrc/node_modules/resolve-from"
    },
    {
      "action": "add",
      "name": "import-fresh",
      "version": "3.3.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@eslint/eslintrc/node_modules/import-fresh"
    },
    {
      "action": "add",
      "name": "type-fest",
      "version": "0.8.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@eslint/eslintrc/node_modules/type-fest"
    },
    {
      "action": "add",
      "name": "globals",
      "version": "12.4.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@eslint/eslintrc/node_modules/globals"
    },
    {
      "action": "add",
      "name": "@types/json5",
      "version": "0.0.29",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@types/json5"
    },
    {
      "action": "add",
      "name": "ms",
      "version": "2.1.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/eslint-plugin/node_modules/ms"
    },
    {
      "action": "add",
      "name": "ms",
      "version": "2.1.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/parser/node_modules/ms"
    },
    {
      "action": "add",
      "name": "@typescript-eslint/types",
      "version": "4.15.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/types"
    },
    {
      "action": "add",
      "name": "@nodelib/fs.stat",
      "version": "2.0.4",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/typescript-estree/node_modules/@nodelib/fs.stat"
    },
    {
      "action": "add",
      "name": "array-union",
      "version": "2.1.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/typescript-estree/node_modules/array-union"
    },
    {
      "action": "add",
      "name": "ignore",
      "version": "5.1.8",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/typescript-estree/node_modules/ignore"
    },
    {
      "action": "add",
      "name": "micromatch",
      "version": "4.0.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/typescript-estree/node_modules/micromatch"
    },
    {
      "action": "add",
      "name": "fast-glob",
      "version": "3.2.5",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/typescript-estree/node_modules/fast-glob"
    },
    {
      "action": "add",
      "name": "path-type",
      "version": "4.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/typescript-estree/node_modules/path-type"
    },
    {
      "action": "add",
      "name": "dir-glob",
      "version": "3.0.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/typescript-estree/node_modules/dir-glob"
    },
    {
      "action": "add",
      "name": "slash",
      "version": "3.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/typescript-estree/node_modules/slash"
    },
    {
      "action": "add",
      "name": "globby",
      "version": "11.0.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/typescript-estree/node_modules/globby"
    },
    {
      "action": "add",
      "name": "ansi-colors",
      "version": "4.1.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/enquirer/node_modules/ansi-colors"
    },
    {
      "action": "add",
      "name": "json5",
      "version": "2.2.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-loader/node_modules/json5"
    },
    {
      "action": "add",
      "name": "eslint-visitor-keys",
      "version": "1.3.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-utils/node_modules/eslint-visitor-keys"
    },
    {
      "action": "add",
      "name": "@typescript-eslint/visitor-keys",
      "version": "4.15.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/visitor-keys"
    },
    {
      "action": "add",
      "name": "@typescript-eslint/scope-manager",
      "version": "4.15.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/scope-manager"
    },
    {
      "action": "add",
      "name": "color-name",
      "version": "1.1.4",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/color-name"
    },
    {
      "action": "add",
      "name": "color-convert",
      "version": "2.0.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/color-convert"
    },
    {
      "action": "add",
      "name": "ansi-styles",
      "version": "4.3.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/ansi-styles"
    },
    {
      "action": "add",
      "name": "has-flag",
      "version": "4.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/has-flag"
    },
    {
      "action": "add",
      "name": "prelude-ls",
      "version": "1.2.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/prelude-ls"
    },
    {
      "action": "add",
      "name": "supports-color",
      "version": "7.2.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/supports-color"
    },
    {
      "action": "add",
      "name": "type-check",
      "version": "0.4.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/type-check"
    },
    {
      "action": "add",
      "name": "levn",
      "version": "0.4.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/levn"
    },
    {
      "action": "add",
      "name": "eslint-visitor-keys",
      "version": "1.3.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/espree/node_modules/eslint-visitor-keys"
    },
    {
      "action": "add",
      "name": "language-subtag-registry",
      "version": "0.3.21",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/language-subtag-registry"
    },
    {
      "action": "add",
      "name": "astral-regex",
      "version": "2.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/table/node_modules/astral-regex"
    },
    {
      "action": "add",
      "name": "color-name",
      "version": "1.1.4",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/table/node_modules/color-name"
    },
    {
      "action": "add",
      "name": "color-convert",
      "version": "2.0.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/table/node_modules/color-convert"
    },
    {
      "action": "add",
      "name": "ansi-styles",
      "version": "4.3.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/table/node_modules/ansi-styles"
    },
    {
      "action": "add",
      "name": "json-schema-traverse",
      "version": "1.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/table/node_modules/json-schema-traverse"
    },
    {
      "action": "add",
      "name": "ajv",
      "version": "7.1.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/table/node_modules/ajv"
    },
    {
      "action": "add",
      "name": "strip-bom",
      "version": "3.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/tsconfig-paths/node_modules/strip-bom"
    },
    {
      "action": "add",
      "name": "@eslint/eslintrc",
      "version": "0.3.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@eslint/eslintrc"
    },
    {
      "action": "add",
      "name": "debug",
      "version": "4.3.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/eslint-plugin/node_modules/debug"
    },
    {
      "action": "add",
      "name": "semver",
      "version": "7.3.4",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/eslint-plugin/node_modules/semver"
    },
    {
      "action": "add",
      "name": "debug",
      "version": "4.3.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/parser/node_modules/debug"
    },
    {
      "action": "add",
      "name": "array.prototype.flatmap",
      "version": "1.2.4",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/array.prototype.flatmap"
    },
    {
      "action": "add",
      "name": "eslint-visitor-keys",
      "version": "1.3.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/babel-eslint/node_modules/eslint-visitor-keys"
    },
    {
      "action": "add",
      "name": "enquirer",
      "version": "2.3.6",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/enquirer"
    },
    {
      "action": "add",
      "name": "loader-utils",
      "version": "2.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-loader/node_modules/loader-utils"
    },
    {
      "action": "add",
      "name": "resolve",
      "version": "1.20.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-plugin-import/node_modules/resolve"
    },
    {
      "action": "add",
      "name": "axe-core",
      "version": "4.1.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-plugin-jsx-a11y/node_modules/axe-core"
    },
    {
      "action": "add",
      "name": "@babel/code-frame",
      "version": "7.12.11",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/@babel/code-frame"
    },
    {
      "action": "add",
      "name": "chalk",
      "version": "4.1.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/chalk"
    },
    {
      "action": "add",
      "name": "optionator",
      "version": "0.9.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/optionator"
    },
    {
      "action": "add",
      "name": "semver",
      "version": "7.3.4",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/semver"
    },
    {
      "action": "add",
      "name": "language-tags",
      "version": "1.0.5",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/language-tags"
    },
    {
      "action": "add",
      "name": "string-natural-compare",
      "version": "3.0.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/string-natural-compare"
    },
    {
      "action": "add",
      "name": "tsconfig-paths",
      "version": "3.9.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/tsconfig-paths"
    }
  ],
  "removed": [
    {
      "action": "remove",
      "name": "@types/eslint-visitor-keys",
      "version": "1.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@types/eslint-visitor-keys"
    },
    {
      "action": "remove",
      "name": "commander",
      "version": "2.20.3",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-plugin-jsx-a11y/node_modules/commander"
    },
    {
      "action": "remove",
      "name": "aria-query",
      "version": "3.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-plugin-jsx-a11y/node_modules/aria-query"
    },
    {
      "action": "remove",
      "name": "ansi-regex",
      "version": "4.1.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/ansi-regex"
    },
    {
      "action": "remove",
      "name": "semver",
      "version": "5.7.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/cross-spawn/node_modules/semver"
    },
    {
      "action": "remove",
      "name": "eslint-utils",
      "version": "1.4.3",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/eslint-utils"
    },
    {
      "action": "remove",
      "name": "path-key",
      "version": "2.0.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/path-key"
    },
    {
      "action": "remove",
      "name": "regexpp",
      "version": "2.0.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/regexpp"
    },
    {
      "action": "remove",
      "name": "shebang-regex",
      "version": "1.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/shebang-regex"
    },
    {
      "action": "remove",
      "name": "shebang-command",
      "version": "1.2.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/shebang-command"
    },
    {
      "action": "remove",
      "name": "strip-ansi",
      "version": "5.2.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/strip-ansi"
    },
    {
      "action": "remove",
      "name": "which",
      "version": "1.3.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/which"
    },
    {
      "action": "remove",
      "name": "rimraf",
      "version": "2.6.3",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/flat-cache/node_modules/rimraf"
    },
    {
      "action": "remove",
      "name": "get-stdin",
      "version": "6.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/get-stdin"
    },
    {
      "action": "remove",
      "name": "path-exists",
      "version": "2.1.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/loader-fs-cache/node_modules/path-exists"
    },
    {
      "action": "remove",
      "name": "find-up",
      "version": "1.1.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/loader-fs-cache/node_modules/find-up"
    },
    {
      "action": "remove",
      "name": "pkg-dir",
      "version": "1.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/loader-fs-cache/node_modules/pkg-dir"
    },
    {
      "action": "remove",
      "name": "find-cache-dir",
      "version": "0.1.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/loader-fs-cache/node_modules/find-cache-dir"
    },
    {
      "action": "remove",
      "name": "loader-fs-cache",
      "version": "1.0.3",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/loader-fs-cache"
    },
    {
      "action": "remove",
      "name": "ansi-regex",
      "version": "4.1.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/table/node_modules/ansi-regex"
    },
    {
      "action": "remove",
      "name": "emoji-regex",
      "version": "7.0.3",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/table/node_modules/emoji-regex"
    },
    {
      "action": "remove",
      "name": "is-fullwidth-code-point",
      "version": "2.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/table/node_modules/is-fullwidth-code-point"
    },
    {
      "action": "remove",
      "name": "strip-ansi",
      "version": "5.2.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/table/node_modules/strip-ansi"
    },
    {
      "action": "remove",
      "name": "string-width",
      "version": "3.1.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/table/node_modules/string-width"
    },
    {
      "action": "remove",
      "name": "write",
      "version": "1.0.3",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/write"
    },
    {
      "action": "remove",
      "name": "xregexp",
      "version": "4.4.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/xregexp"
    }
  ],
  "updated": [
    {
      "action": "update",
      "name": "array-includes",
      "version": "3.1.3",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/array-includes",
      "previousVersion": "3.1.2"
    },
    {
      "action": "update",
      "name": "eslint-visitor-keys",
      "version": "2.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-visitor-keys",
      "previousVersion": "1.3.0"
    },
    {
      "action": "update",
      "name": "@typescript-eslint/typescript-estree",
      "version": "4.15.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/typescript-estree",
      "previousVersion": "2.34.0"
    },
    {
      "action": "update",
      "name": "espree",
      "version": "7.3.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/espree",
      "previousVersion": "6.2.1"
    },
    {
      "action": "update",
      "name": "flatted",
      "version": "3.1.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/flatted",
      "previousVersion": "2.0.2"
    },
    {
      "action": "update",
      "name": "slice-ansi",
      "version": "4.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/table/node_modules/slice-ansi",
      "previousVersion": "2.1.0"
    },
    {
      "action": "update",
      "name": "flat-cache",
      "version": "3.0.4",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/flat-cache",
      "previousVersion": "2.0.1"
    },
    {
      "action": "update",
      "name": "@typescript-eslint/experimental-utils",
      "version": "4.15.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/experimental-utils",
      "previousVersion": "2.34.0"
    },
    {
      "action": "update",
      "name": "@typescript-eslint/eslint-plugin",
      "version": "4.15.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/eslint-plugin",
      "previousVersion": "2.34.0"
    },
    {
      "action": "update",
      "name": "@typescript-eslint/parser",
      "version": "4.15.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/@typescript-eslint/parser",
      "previousVersion": "2.34.0"
    },
    {
      "action": "update",
      "name": "babel-eslint",
      "version": "10.1.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/babel-eslint",
      "previousVersion": "10.1.0"
    },
    {
      "action": "update",
      "name": "eslint-config-airbnb",
      "version": "18.2.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-config-airbnb",
      "previousVersion": "18.2.1"
    },
    {
      "action": "update",
      "name": "eslint-config-prettier",
      "version": "8.1.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-config-prettier",
      "previousVersion": "6.15.0"
    },
    {
      "action": "update",
      "name": "eslint-config-react-app",
      "version": "6.0.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-config-react-app",
      "previousVersion": "5.2.1"
    },
    {
      "action": "update",
      "name": "eslint-loader",
      "version": "4.0.2",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-loader",
      "previousVersion": "3.0.3"
    },
    {
      "action": "update",
      "name": "emoji-regex",
      "version": "9.2.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-plugin-jsx-a11y/node_modules/emoji-regex",
      "previousVersion": "7.0.3"
    },
    {
      "action": "update",
      "name": "eslint-plugin-prettier",
      "version": "3.3.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-plugin-prettier",
      "previousVersion": "3.3.1"
    },
    {
      "action": "update",
      "name": "eslint-plugin-react-hooks",
      "version": "4.2.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-plugin-react-hooks",
      "previousVersion": "1.7.0"
    },
    {
      "action": "update",
      "name": "resolve",
      "version": "1.20.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-plugin-react/node_modules/resolve",
      "previousVersion": "1.19.0"
    },
    {
      "action": "update",
      "name": "cross-spawn",
      "version": "7.0.3",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint/node_modules/cross-spawn",
      "previousVersion": "6.0.5"
    },
    {
      "action": "update",
      "name": "esquery",
      "version": "1.4.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/esquery",
      "previousVersion": "1.3.1"
    },
    {
      "action": "update",
      "name": "file-entry-cache",
      "version": "6.0.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/file-entry-cache",
      "previousVersion": "5.0.1"
    },
    {
      "action": "update",
      "name": "jsx-ast-utils",
      "version": "3.2.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/jsx-ast-utils",
      "previousVersion": "2.4.1"
    },
    {
      "action": "update",
      "name": "eslint-plugin-jsx-a11y",
      "version": "6.4.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-plugin-jsx-a11y",
      "previousVersion": "6.2.3"
    },
    {
      "action": "update",
      "name": "object.fromentries",
      "version": "2.0.4",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/object.fromentries",
      "previousVersion": "2.0.3"
    },
    {
      "action": "update",
      "name": "prettier",
      "version": "2.2.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/prettier",
      "previousVersion": "2.2.1"
    },
    {
      "action": "update",
      "name": "eslint-plugin-flowtype",
      "version": "5.3.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-plugin-flowtype",
      "previousVersion": "4.6.0"
    },
    {
      "action": "update",
      "name": "string.prototype.matchall",
      "version": "4.0.4",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/string.prototype.matchall",
      "previousVersion": "4.0.3"
    },
    {
      "action": "update",
      "name": "eslint-plugin-react",
      "version": "7.22.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-plugin-react",
      "previousVersion": "7.19.0"
    },
    {
      "action": "update",
      "name": "table",
      "version": "6.0.7",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/table",
      "previousVersion": "5.4.6"
    },
    {
      "action": "update",
      "name": "eslint",
      "version": "7.20.0",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint",
      "previousVersion": "6.8.0"
    },
    {
      "action": "update",
      "name": "eslint-plugin-import",
      "version": "2.22.1",
      "path": "/Volumes/Development/nciocpl/clinical-trials-listing-app/node_modules/eslint-plugin-import",
      "previousVersion": "2.20.1"
    }
  ],
  "moved": [],
  "failed": [],
  "warnings": [
    "tsutils@3.20.0 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself."
  ],
  "audit": {
    "actions": [],
    "advisories": {
      "1603": {
        "findings": [
          {
            "version": "1.10.0",
            "paths": [
              "react-dev-utils>immer"
            ]
          }
        ],
        "id": 1603,
        "created": "2021-02-19T18:18:20.058Z",
        "updated": "2021-02-19T18:18:32.751Z",
        "deleted": null,
        "title": "Prototype Pollution",
        "found_by": {
          "link": "",
          "name": "Anonymous",
          "email": ""
        },
        "reported_by": {
          "link": "",
          "name": "Anonymous",
          "email": ""
        },
        "module_name": "immer",
        "cves": [
          "CVE-2020-28477"
        ],
        "vulnerable_versions": "<8.0.1",
        "patched_versions": ">=8.0.1",
        "overview": "## Overview\n\nAffected versions of `immer` are vulnerable to Prototype Pollution.\n\n## Proof of exploit\n\n```\nconst {applyPatches, enablePatches} = require(\"immer\");\nenablePatches();\nlet obj = {};\nconsole.log(\"Before : \" + obj.polluted);\napplyPatches({}, [ { op: 'add', path: [ \"__proto__\", \"polluted\" ], value: \"yes\" } ]);\n// applyPatches({}, [ { op: 'replace', path: [ \"__proto__\", \"polluted\" ], value: \"yes\" } ]);\nconsole.log(\"After : \" + obj.polluted);\n```\n\n## Remediation\n\nVersion 8.0.1 contains a [fix](https://github.com/immerjs/immer/commit/da2bd4fa0edc9335543089fe7d290d6a346c40c5) for this vulnerability, updating is recommended.",
        "recommendation": "Upgrade to version 8.0.1 or later",
        "references": "- [GitHub Advisory](https://github.com/advisories/GHSA-9qmh-276g-x5pj)\n",
        "access": "public",
        "severity": "high",
        "cwe": "CWE-1321",
        "metadata": {
          "module_type": "",
          "exploitability": 8,
          "affected_components": ""
        },
        "url": "https://npmjs.com/advisories/1603"
      }
    },
    "muted": [],
    "metadata": {
      "vulnerabilities": {
        "info": 0,
        "low": 0,
        "moderate": 0,
        "high": 1,
        "critical": 0
      },
      "dependencies": 319,
      "devDependencies": 1929,
      "optionalDependencies": 46,
      "totalDependencies": 2249
    }
  },
  "funding": "106 packages are looking for funding",
  "elapsed": 11590
}
```

**Comment by arcepaul** [comment]
Fixed via PR #34
